aten/src/ATen/core/op_registration/op_registration_test.cpp

```diff
@@ -2241,6 +2241,22 @@ TEST(NewOperatorRegistrationTest, testDelayedListener) {
   EXPECT_EQ(initial_num_deregisters + 1, listener_ptr->num_deregisters_);
 }
 
+TEST(NewOperatorRegistrationTest, testImplNoDefGetsCaught) {
+  auto danglingImpls = Dispatcher::singleton().findDanglingImpls();
+  std::string error_str = "Discovered operators that have been registered through the dispatcher"
+                          " without explicitly specifying their schemas. Please do so using"
+                          " the TORCH_LIBRARY macro. Suspect operators:\n";
+  for (auto& op : danglingImpls) {
+      auto& op_name = op.operator_name();
+      error_str += "\t" + op_name.name;
+      if (op_name.overload_name != "") {
+          error_str += "." + op_name.overload_name;
+      }
+      error_str += "\n";
+  }
+  ASSERT_EQ(danglingImpls.size(), 0) << error_str;
```

Comment:
This test is actually failing already on 13 operators. I'm going to dig more into why / whether or not they're real errors
![image](https://user-images.githubusercontent.com/16311747/96295781-31e08c80-0fbc-11eb-87f6-ee77e2adea08.png)

Reply:
Well, good thing you're doing this PR 👍